2019-11-06 20:05:10,920 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched - 01/app/__init__.py:52]
2019-11-07 22:54:34,493 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched - 01/app/__init__.py:54]
2019-11-10 14:31:55,134 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched - 01/app/__init__.py:56]
2019-11-10 14:41:15,871 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched - 01/app/__init__.py:56]
2019-11-10 16:55:06,052 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched - 01/app/__init__.py:56]
2019-11-10 17:01:56,388 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched - 01/app/__init__.py:56]
2019-11-10 17:03:32,726 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched - 01/app/__init__.py:56]
2019-11-10 17:06:34,537 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched - 01/app/__init__.py:56]
2019-11-10 17:34:29,038 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched - 01/app/__init__.py:56]
2019-11-10 17:54:39,489 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched - 01/app/__init__.py:56]
2019-11-10 17:54:52,803 ERROR: Exception on /search [GET] [in /home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/donjoe/Desktop/projects/sched - 01/app/main/routes.py", line 148, in search
    next_url=next_url, prev_url=prev_url)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/donjoe/Desktop/projects/sched - 01/app/templates/search.html", line 4, in template
    <h1>{{ Search Results }}</h1>
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 876, in subparse
    self.stream.expect('variable_end')
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/lexer.py", line 384, in expect
    self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got 'Results'
2019-11-10 17:57:14,046 INFO: Microblog startup [in /home/donjoe/Desktop/projects/sched - 01/app/__init__.py:56]
2019-11-10 17:57:26,711 ERROR: Exception on /search [GET] [in /home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/donjoe/Desktop/projects/sched - 01/app/main/routes.py", line 148, in search
    next_url=next_url, prev_url=prev_url)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/donjoe/Desktop/projects/sched - 01/app/templates/search.html", line 4, in template
    <h1>{{ Search Results }}</h1>
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/parser.py", line 876, in subparse
    self.stream.expect('variable_end')
  File "/home/donjoe/Desktop/projects/sched/venv/lib/python3.5/site-packages/jinja2/lexer.py", line 384, in expect
    self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got 'Results'
